README.md

```diff
@@ -92,47 +92,82 @@ The milestone needs testing on all platforms. Please report issues when found.
     - D3 Open Source compiles level scripts in their own hogfiles. Make sure you copy and overwrite `d3-{platform}.hog`.
 
 ## Building
-Build steps below assume you have already cloned the repository and entered it locally.
-
 #### Building - Windows
-Requires Visual Studio 2022 and C++ Tools (cmake and vcpkg)
+1. Make sure that you have Git and Visual Studio 2022 with the “Desktop development with C++” workload. If you don’t already have those installed or you aren’t sure, then open an elevated Command Prompt and run:
 
-Install and configure vcpkg:
-```sh
-git clone https://github.com/microsoft/vcpkg.git
-cd vcpkg && bootstrap-vcpkg.bat
-setx VCPKG_ROOT="C:\path\to\vcpkg"
-setx PATH=%VCPKG_ROOT%;%PATH%
-```
+    <!--
+    The following code block specifies the full path to the Visual Studio Installer because the Visual Studio Installer doesn’t add itself to the user’s Path. The installer is guaranteed to be in a specific location on 64-bit systems [1]. The installer will be in a different location on 32-bit systems [2], but Visual Studio 2022 doesn’t support 32-bit systems [3] so we can ignore that detail.
 
-Build Descent 3 using the "x86 native tools command prompt"
-```sh
-cmake --preset win32 -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset win32 --config [Debug|Release]
-```
+    [1]: <https://learn.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2022>
+    [2]: <https://github.com/microsoft/vswhere/wiki#installing>
+    [3]: <https://learn.microsoft.com/en-us/answers/questions/1689898/does-visual-studio-build-tools-2022-support-32-bit>
+    -->
 
-#### Building - MacOS
-```sh
-brew bundle install
-cmake --preset mac -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset mac --config [Debug|Release]
-```
+    ```batch
+    winget install Git.Git Microsoft.VisualStudio.2022.Community
+
+    "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\setup.exe" modify^
+        --passive^
+        --channelId VisualStudio.17.Release^
+        --productId Microsoft.VisualStudio.Product.Community^
+        --add Microsoft.VisualStudio.Workload.NativeDesktop;includeRecommended
+    ```
+
+2. Open a “x86 Native Tools Command Prompt” and run:
+
+    ```batch
+    git clone https://github.com/DescentDevelopers/Descent3
+    cd Descent3
```

Comment:
Now that I see it, I wonder if these 2 lines should go under the headline "Building", with something along the lines of "First, clone the Descent 3 repository: ... Then, follow platform-specific compilation instructions". What do you think about that? Good job otherwise, I really like these new instructions

Reply:
> Now that I see it, I wonder if these 2 lines should go under the headline "Building", with something along the lines of "First, clone the Descent 3 repository: ... Then, follow platform-specific compilation instructions". What do you think about that?

OK, so I just tried changing the instructions to work that way, and [I don’t like how they look](https://github.com/Jayman2000/Descent3-pr/tree/532505f3f2e6f058c8dfdbf06e5fead4ba55d266?tab=readme-ov-file#building). At the moment, the instructions tell you to install prerequisites, clone the repository and then run CMake. In order to move the cloning step to the beginning, I had to tell users to install some prerequisites, then clone the repo, then install more prerequisites and then run CMake. I think that the instructions flow better when the steps for installing prerequisites are grouped together. Additionally, moving the cloning step to the beginning resulted in Windows users having to use four different Command Prompt windows instead of two.

I suppose that we could skip the “make sure that Git is installed” part, but then the instructions would fail for users that don’t already have the `git` command on their path. For example, if a user is on Windows and they only have [GitHub Desktop](https://desktop.github.com) installed, then the `git` command won’t be on their path. Additionally, Windows users would still end up using three different Command Prompt windows even if we removed that part. They would use a regular Command Prompt in order to clone the repo, an administrator Command Prompt in order to install Visual Studio 2022 and the C++ workload, and an x86 tools Command Prompt in order to run CMake.

> Good job otherwise, I really like these new instructions

Thank you.

Reply:
Indeed I also prefer the previous version, this one is more cluttered than I thought. Let's revert this last change and merge the PR

Reply:
Does this include the new ninja generator that windows builds use? I don't think VS includes it by default. I could be blind here but I'm not seeing it in the instructions.

Reply:
Isn't ninja shipped with Visual Studio when you install CMake nowadays?

Reply:
A second opinion would be nice, but when I reinstalled VS it didn't come with ninja--at least the cmake preset wouldn't take.

Reply:
> Does this include the new ninja generator that windows builds use? I don't think VS includes it by default. I could be blind here but I'm not seeing it in the instructions.

I had a similar concern when I was rebasing this PR after #422 was merged. I had considered doing this:

```diff
diff --git a/README.md b/README.md
index 41aee876..ae79ca5b 100644
--- a/README.md
+++ b/README.md
@@ -104,7 +104,7 @@ The milestone needs testing on all platforms. Please report issues when found.
     -->

     ```batch
-    winget install Git.Git Microsoft.VisualStudio.2022.Community
+    winget install Git.Git Microsoft.VisualStudio.2022.Community Ninja-build.Ninja

     "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\setup.exe" modify^
         --passive^
```

But after testing, I found that it wasn’t necessary.

---

> Isn't ninja shipped with Visual Studio when you install CMake nowadays?

I think that this is correct. I wasn’t able to find any documentation that mentions that Ninja comes with Visual Studio’s CMake component, but I was able to verify that it comes with Visual Studio’s CMake component by doing the following:

1. Make sure that CMake and Ninja aren’t installed outside of Visual Studio:

    ```batch
     winget uninstall Kitware.CMake Ninja-build.Ninja
    ```

2. Uninstall the Visual Studio Installer.

3. Install Visual Studio Build Tools 2022 and the “C++ CMake tools for Windows” component. To do so, open an elevated Command Prompt and run:

    ```batch
    winget install Microsoft.VisualStudio.2022.BuildTools

    "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\setup.exe" modify^
        --passive^
        --channelId VisualStudio.17.Release^
        --productId Microsoft.VisualStudio.Product.BuildTools^
        --add Microsoft.VisualStudio.Component.VC.CMake.Project
    ```

4. Open “x86 Native Tools Command Prompt for VS 2022”.

5. Make sure that Ninja is installed:

    ```batch
    where ninja
    ```

---

> I receive the following when using a fresh install of VS 2022 community per instructions, along with C++ desktop tools.
> 
> ```
> [cmake] CMake Error: CMake was unable to find a build program corresponding to "Ninja Multi-Config".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
> [cmake] CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
> [cmake] CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
> [cmake] -- Configuring incomplete, errors occurred!
> ```

I can’t reproduce that error. The best that I can think of is that somehow Ninja isn’t ending up on your `Path`. What happens when you run these commands?

```batch
where ninja
set Path
```